src/requests/models.py

```diff
@@ -95,7 +95,7 @@ def path_url(self):
             path = "/"
 
         url.append(path)
-
+        url.doesnotexist(path)
```

Comment:
**Suggestion:** The 'url' object does not have a 'doesnotexist' method. This will raise an AttributeError.
```suggestion
        # Assuming you wanted to append the path to the url
        url.append(path)
```